product_docs/docs/edbcloud/beta/getting_started/01_preparing_azure/01_understanding_qotas_in_azure.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: "Understanding resource requirements"
```

Comment:
I like the basic structure here but this page isn't exactly about "resource requirements" - it's about BigAnimal requirements in Azure more generally.

Reply:
Changed to: Understanding requirements in Azure